content/en/blog/_posts/2024-08-15-moving-cgroup-v1-support-maintenance-mode-kubernetes-1-31.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: blog
+title: "Moving cgroup v1 Support into Maintenance Mode in Kubernetes 1.31"
+date: 2024-08-15
+slug: moving-cgroup-v1-support-maintenance-mode-kubernetes-1-31
+author: Harshal Patil
+---
+
+As Kubernetes continues to evolve and adapt to the changing landscape of
+container orchestration, the community has decided to move cgroup v1 support
+into maintenance mode in v1.31. This shift aligns with the broader industry's
```

Comment:
Reading this, I think "what is maintenance mode"? Maybe we can link to the markdown as we define it later on in the document.

Reply:
Updated, thanks.